test/api_integration/apis/saved_objects_management/relationships.ts

```diff
@@ -342,7 +342,7 @@ export default function ({ getService }: FtrProviderContext) {
     });
 
     describe('index patterns', () => {
-      it('should validate visualization response schema', async () => {
+      it('should validate index-pattern response schema', async () => {
```

Comment:
unrelated to this PR:
Asserts against an index-pattern response schema, not a visualization.